concepticondata/conceptlists/Thwaites-2020-98.tsv

```diff
@@ -0,0 +1,99 @@
+ID	NUMBER	ENGLISH	TRANSITIVITY	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Thwaites-2020-98-1	1	abduct	transitive		
+Thwaites-2020-98-2	2	acquire	transitive	3892	ACQUIRE (SOMETHING)
+Thwaites-2020-98-3	3	afford	transitive	3893	AFFORD
+Thwaites-2020-98-4	4	attach	transitive	3894	ATTACH
+Thwaites-2020-98-5	5	avoid	transitive	3639	AVOID
+Thwaites-2020-98-6	6	baffle	transitive		
+Thwaites-2020-98-7	7	bury	transitive	1719	BURY
+Thwaites-2020-98-8	8	carry	transitive	700	CARRY
+Thwaites-2020-98-9	9	chuck	transitive		
+Thwaites-2020-98-10	10	crush	transitive	1520	CRUSH
+Thwaites-2020-98-11	11	deduct	transitive		
+Thwaites-2020-98-12	12	defy	transitive		
+Thwaites-2020-98-13	13	detect	transitive	3895	DETECT
+Thwaites-2020-98-14	14	devour	transitive		
+Thwaites-2020-98-15	15	dilute	transitive	550	DILUTE
+Thwaites-2020-98-16	16	dissect	transitive		
+Thwaites-2020-98-17	17	eject	transitive		
+Thwaites-2020-98-18	18	eliminate	transitive		
+Thwaites-2020-98-19	19	enhance	transitive	2872	IMPROVE (SOMETHING)
+Thwaites-2020-98-20	20	expedite	transitive		
+Thwaites-2020-98-21	21	expel	transitive		
+Thwaites-2020-98-22	22	extinguish	transitive	127	EXTINGUISH
+Thwaites-2020-98-23	23	furnish	transitive		
+Thwaites-2020-98-24	24	gratify	transitive	1271	DELIGHT
+Thwaites-2020-98-25	25	illuminate	transitive		
+Thwaites-2020-98-26	26	inject	transitive		
+Thwaites-2020-98-27	27	injure	transitive	44	INJURE
+Thwaites-2020-98-28	28	irrigate	transitive	3078	IRRIGATE
+Thwaites-2020-98-29	29	knead	transitive	274	KNEAD
+Thwaites-2020-98-30	30	lick	transitive	319	LICK
+Thwaites-2020-98-31	31	nourish	transitive		
+Thwaites-2020-98-32	32	pacify	transitive		
+Thwaites-2020-98-33	33	pollinate	transitive		
+Thwaites-2020-98-34	34	promote	transitive		
+Thwaites-2020-98-35	35	propel	transitive		
+Thwaites-2020-98-36	36	raise	transitive	1317	RAISE
```

Comment:
We have "to raise, lift (tr.)" mapped to LIFT in a couple of lists and other forms of "raise" are mapped to RAISE (BRING UP). Is there maybe more information on the words in the study?

Reply:
Unfortunately, it's not specified. I've also checked now against the SUBTLES_UK database where it was taken from. So I'd rather unmap it then.